.clippy.toml

```diff
@@ -0,0 +1,3 @@
+# This feature is in Rust nightly, as of v1.62. The line is added in anticipation.
+# Once the feature stabilizes, remove the #[allow(clippy::unwrap_used)] lines in front of every `mod test`
+allow-unwrap-in-tests = true
```

Comment:
add an extra line to remove 
![image](https://github.com/user-attachments/assets/84971e82-7f57-4e37-878c-55772114380a)